contrib/lkt/language/parser.py

```diff
@@ -470,6 +470,28 @@ def first_no_paren_parent():
             Entity.parent
         )
 
+    @langkit_property(public=True, return_type=T.LKNode.entity)
+    def topmost_invalid_decl(
+        topmost=(T.LKNode.entity, No(T.LKNode.entity))
+    ):
+        """
+        Return the topmost FullDecl annotated with ``@invalid``,
+        null otherwise.
```

Comment:
Can you document what the `topmost` argument is? Also clarify whether the current node is included in the search (it looks like it’s not looking at the implementation, but that should be documented here).